README.md

```diff
@@ -42,6 +46,66 @@ immutad●t comes with a large set of built-in utilities, mostly based on [lodas
 
 If you are already familiar with [lodash](https://lodash.com/) and [ES2015+](https://github.com/tc39/ecma262#ecmascript) then you should be able to use immutad●t quickly.
 
+## Installation
+
+immutad●t is available on [npm repository](https://www.npmjs.com/package/immutadot).
+
+using npm:
+
+```shell
+$ npm install immutadot
+```
+
+using yarn:
+
+```shell
+$ yarn add immutadot
+```
+
+or you can directly download [sources](https://github.com/Zenika/immutadot/releases).
+
+## Usage
+
+in browser:
+
+```js
+  import { push } from 'immutadot'
+
+  push({ nested: { prop: [1, 2] } }, 'nested.prop', 3, 4)
+  // → { nested: { prop: [1, 2, 3, 4] } }
+```
+
+in node:
+
+```js
+  const { push } = require('immutadot')
+
+  push({ nested: { prop: [1, 2] } }, 'nested.prop', 3, 4)
+  // → { nested: { prop: [1, 2, 3, 4] } }
+```
+
+Feel free to try immutad●t with our online [demo](https://npm.runkit.com/immutadot).
+
+## Documentation
+
+You can find out a full list of immutad●t's features in our [documentation](https://zenika.github.io/immutadot).
+
+## Contributing
+
+We want contributing to immutad●t to be fun, enjoyable, and educational for anyone, and everyone.
+
+### [Code of Conduct](https://github.com/Zenika/immutadot/blob/master/.github/CODE_OF_CONDUCT.md)
+
+In the interest of fostering an open and welcoming environment, we have adopted a Code of Conduct that we expect project participants to adhere to. Please read the [full text](https://github.com/Zenika/immutadot/blob/master/.github/CODE_OF_CONDUCT.md) so that you can understand what actions will and will not be tolerated.
```

Comment:
Please read the full text so you can understand what behaviour will and will not be tolerated.